assets/quick-order-list.js

```diff
@@ -284,9 +284,10 @@ class QuickOrderList extends HTMLElement {
         } else {
           this.updateMessage(-parseInt(quantityElement.dataset.cartQuantity))
         }
-      }).catch(() => {
+      }).catch((error) => {
```

Comment:
Kai suggested to keep it, so next time we will know right away what’s causing the error.

Reply:
We dont really add this in other places 🤔 Shouldnt we just add it if we are debugging?

Reply:
We can keep the logging. We actually log errors to the console in a few places in the theme (search for `console.error`). 

In normal circumstances, we should never see any errors logged, but when making changes it's probably better to see an error in the console and catch it early rather than missing a suppressed error.